### PR TITLE
New null rod variant: Golden crucifix

### PR DESCRIPTION
**Pull request by ynot01**
# Document the changes in your pull request

Does ranged AOE damage instead of melee damage

Range

Effects, victims also get large text "The holy light burns you!" every 0.8 seconds

# Inspiration: 

FAITH

image

# Spriting
Item sprite by OrcaCora and Cuackles
image

# Wiki Documentation

Has 0 force and 0 throw force

When used in hand, every mob in an area in front of you will be constantly damaged

Moving will halt the effect

Damage is ~5 burn and ~15 stamina per second

Unholy creatures (cult, wizard, vampire, heretic, etc) will instead be damaged for ~10 burn and ~10 stamina per second

# Changelog

:cl:  ynot01, OrcaCora, Cuackles
rscadd: Added the golden crucifix null rod variant
/:cl:

### CODE REVIEW

**Comment by Rdrazga** [comment]
Now we just need to give the chaplain an exorcism ritual that they can do using this that deconverts rather then just filling people full of holy water

**Comment by Simplehorror** [comment]
add garlic from vampire survivors

**Comment by Simplehorror** [comment]
also i love this

**Comment by cowbot92** [comment]
Have FAITH

**Comment by Moltijoe** [comment]
make dying with it in your hand play the mortis sound effect

**Comment by Moltijoe** [comment]
does this let chaplain attack through walls?

**Comment by ynot01** [comment]
> does this let chaplain attack through walls?

Shouldn't

File: code/game/objects/items/holy_weapons.dm [hunk 1]
```diff
@@ -869,3 +869,84 @@
...
+	held_up = TRUE
+	w_class = WEIGHT_CLASS_GIGANTIC // Heavy, huh?
+	slot_flags = 0
+	holy_glow_fx = mutable_appearance('icons/effects/genetics.dmi', "servitude", -MUTATIONS_LAYER)
+	user.add_overlay(holy_glow_fx)
+	holy_glow_light = user.mob_light(_color = LIGHT_COLOR_HOLY_MAGIC, _range = 2)
+	RegisterSignal(user, COMSIG_MOVABLE_MOVED, .proc/unwield)
+	START_PROCESSING(SSfastprocess, src)
+
+/obj/item/nullrod/cross/dropped(mob/user)
```
Comment (ToasterBiome) [inline code/game/objects/items/holy_weapons.dm]: we have a signal for this, COMSIG_ITEM_DROPPED

File: code/modules/spells/spell_types/cone_spells.dm [hunk 2]
```diff
@@ -106,6 +106,14 @@
 	else
 		return 2
 
+// Global ver
```
Comment (ToasterBiome) [inline code/modules/spells/spell_types/cone_spells.dm]: unused

**Review by ToasterBiome — CHANGES_REQUESTED** [review]

File: code/game/objects/items/holy_weapons.dm [hunk 3]
```diff
@@ -869,3 +869,82 @@
...
+		return PROCESS_KILL // something has gone terribly wrong
+	
+	var/notify = FALSE
+	if(COOLDOWN_FINISHED(src, holy_notification))
+		COOLDOWN_START(src, holy_notification, 0.8 SECONDS)
+		notify = TRUE
+
+	var/list/chapview = view(4, get_turf(loc))
+	for(var/mob/living/L in view(2, get_teleport_loc(get_turf(loc), loc, 2)))
+		if(!L.mind?.holy_role && (L in chapview)) // Priests are unaffeted, trying to use it as a non-priest will harm you
```
Comment (Xoxeyos) [inline code/game/objects/items/holy_weapons.dm]: unaffeted to unaffected

**Review by cuackles — CHANGES_REQUESTED** [review]
orca and i bealive the item needs the outline a bit more notable/harsher, apart from that all good

**Comment by ynot01** [comment]
> orca and i bealive the item needs the outline a bit more notable/harsher, apart from that all good

on the inhand or the object sprite? and by outline do you mean shadow?

**Comment by cuackles** [comment]
> > orca and i bealive the item needs the outline a bit more notable/harsher, apart from that all good
> 
> on the inhand or the object sprite? and by outline do you mean shadow?

Object, also i didn't interpret it as a shadow Wich it actually makes a lot more sense , in that case it would need an actual outline around it, the inhand is good and can stay as it is

**Comment by KillerOrcaCora** [comment]
What shadow? There is only 2 colors and shadow is determined by light which is nowhere, you can’t have your outline be your light and your outline at the same time if it’s one primary color or else it just leaks out like it’s doing now. If you want to keep the games look just slap a more noticeable outline and make sure to fully cover it.

**Comment by cuackles** [comment]
> What shadow? There is only 2 colors and shadow is determined by light which is nowhere, you can’t have your outline be your light and your outline at the same time if it’s one primary color or else it just leaks out like it’s doing now. If you want to keep the games look just slap a more noticeable outline and make sure to fully cover it.

Orca the darker color is the shadow, it shows thickness in the cross

**Comment by KillerOrcaCora** [comment]
The darker color is the outline, not the shadow. Shadow is inside the sprite not the outside of it clown.

**Comment by cuackles** [comment]
It's not an outline, it's the shadow, there was never an outline we where lied to, the outline was capitalistic propaganda

**Comment by KillerOrcaCora** [comment]
God damnit! AGAIN!

**Comment by cuackles** [comment]
Our spriting team everybody, there are 0 braincells between us

**Comment by KillerOrcaCora** [comment]
OMG THERE IS NO OUTLINE AT ALL IT REALLY WAS THE SHADOW REEEEEEE

**Comment by ynot01** [comment]
I'm going to be honest I don't want to toss a million screenshots back and forth in this PR but given this is the most response I've ever seen from spriting maintainers I'm going to anyways

Tell me if you're satisfied with this (enlarged for clarity)
image

**Comment by KillerOrcaCora** [comment]
Come to #spriting

**Review by cuackles — APPROVED** [review]
✝️

**Pull request merged**